app_identity_service_test.go

```
@@ -0,0 +1,93 @@
+		IsSystemAdmin: false,
+	}
+
+	mux.HandleFunc("/api/v1/appIdentities", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodPost)
+		testAuthorization(t, r)
+		testURLQuery(t, r, "appId", "my-app")
+		testURLQuery(t, r, "type", "TOKEN")
+		testURLQuery(t, r, "isSystemAdmin", "false")
+		w.WriteHeader(http.StatusCreated)
+		w.Header().Set("Location", "/api/v1/appIdentities/my-app")
+		fmt.Fprint(w, `{"appId":"my-app","type":"TOKEN","secret":"appToken-secret","systemAdmin":false}`)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)